addons/website_form/static/src/snippets/s_website_form/000.js

```diff
@@ -193,6 +193,7 @@ odoo.define('website_form.s_website_form', function (require) {
                     }
                     switch (successMode) {
                         case 'redirect':
+                            successPage = successPage.indexOf("/#") === 0 ? successPage.slice(1) : successPage;
```

Comment:
Can't you use `startsWith`?

Reply:
Yes we could, I thought about it but for me it was the same

Reply:
@qsm-odoo I made the change